src/main/resources/hudson/tasks/Mailer/global.jelly

```diff
@@ -32,8 +32,7 @@ THE SOFTWARE.
       <f:textbox />
     </f:entry>
     <f:advanced>
-      <f:optionalBlock name="useSMTPAuth" title="${%Use SMTP Authentication}" checked="${descriptor.smtpAuthUserName!=null}"
-          help="${descriptor.getHelpFile('smtpAuth')}">
```

Comment:
I'm pretty sure that this line is causing a 404 when I look at jenkins/configure...

I'm not as certain about the fix...